### PR TITLE
Policy requirement: Stop using grunt-ts due to dependant vulnerability issues

### PR DESCRIPTION
**Pull request by MSNev**

### CODE REVIEW

File: gruntfile.js [hunk 1]
```diff
@@ -25,21 +25,13 @@ module.exports = function (grunt) {
                 tsconfig: './lib/tsconfig.json'
             },
             dynamicprototest: {
-                tsconfig: './lib/test/tsconfig.json',
-                src: [
-                    './lib/test/Selenium/DynamicProtoTests.ts'
-                ],
-                out: 'lib/test/Selenium/dynamicprototests.js'
+                tsconfig: './lib/test/tsconfig.json'
```
Comment (MSNev) [inline gruntfile.js:28]: These changes also address VSCode pre-checks as the paths within the tsconfig.json are now correct.

The @nevware21/grunt-ts-plugin simplistically just calls ```tsc --project <tsconfig.json>``` while grunt-ts would attempt to create a full command line of the individual files reading some values from tsconfig.json and others from the gruntfile.js

File: rollup/test/tsconfig.json [hunk 2]
```diff
@@ -10,5 +10,6 @@
         "declaration": true,
         "lib": [ "dom", "es2015" ]
     },
-    "files": []
-}
\ No newline at end of file
+    "files": [
```
Comment (xiao-lix) [inline rollup/test/tsconfig.json:13]: I guess it doesn't matter but seems the extra line is unnecessary 😄

File: rollup/tsconfig.json [hunk 3]
```diff
@@ -11,8 +11,7 @@
     "noEmitHelpers": false,
     "alwaysStrict": true,
     "declaration": true,
-    "outDir": "./out",
-    "rootDir": "./",
+    "outDir": "./out/rollup/src",
```
Comment (xiao-lix) [inline rollup/tsconfig.json:14]: Is the outDir not needed here?

File: package.json [hunk 4]
```diff
@@ -52,6 +53,6 @@
     "tslint-config-prettier": "^1.18.0",
     "tslint-microsoft-contrib": "^6.2.0",
     "typedoc": "^0.20.36",
-    "typescript": "^3.7.2"
+    "typescript": "^3.9.9"
```
Comment (xiao-lix) [inline package.json:56]: Just curious, why is typescript not updated to the most recent version?

**Review by xiao-lix — APPROVED** [review]

**Pull request merged**